server/handlers/hosts.go

```diff
@@ -0,0 +1,101 @@
+package handlers
+
+import (
+	"github.com/gofrs/uuid"
+	"github.com/layer5io/meshkit/database"
+	"gorm.io/gorm"
+)
+
+type MesheryHostsPersistor struct {
+	DB *database.Handler
+}
+
+type MesheryHosts struct {
+	ID       uuid.UUID
+	Hostname string
+	Port     int
+}
+type MesheryHostsDisplay struct {
+	ID       uuid.UUID           `json:"id"`
+	Hostname string              `json:"hostname"`
+	Port     int                 `json:"port"`
+	Summary  hostIndivitualCount `json:"summary"`
+}
+type MesheryHostsContextPage struct {
+	Page        int                   `json:"page"`
+	PageSize    int                   `json:"page_size"`
+	Count       int64                 `json:"total_count"`
+	Registrants []MesheryHostsDisplay `json:"registrants"`
+}
+type hostIndivitualCount struct {
+	Count_model        int64
```

Comment:
Generally, we don't use snake case in golang. (you can have a look at other constructs in Meshery for reference).
Please consider using PascalCase

Reply:
Also as I mentioned in the Slack thread, please consider removing `count_` from the keys of this construct as that is evident from the name of this struct 👍